app/code/Magento/Sales/Block/Adminhtml/Order/Create.php

```diff
@@ -60,7 +60,7 @@ protected function _construct()
         $this->buttonList->update('save', 'data_attribute', []);
 
         $this->buttonList->update('save', 'id', 'submit_order_top_button');
-        if ($customerId === null || !$storeId) {
```

Comment:
I would recommend simplifying this condition to:
```
if (!$customerId || !$storeId) {
```

Reply:
I have refactored the if condition

Reply:
Please review https://github.com/magento/magento2/pull/21665/commits/9181dc351fd476e695d9425ae1941ace972c18b7 changes

Reply:
@dmytro-ch  Please review https://github.com/magento/magento2/pull/21665/commits/9181dc351fd476e695d9425ae1941ace972c18b7 changes